README.md

```diff
@@ -2,3 +2,7 @@
 
 This repository contains a set of demos showing many of the tools Knative Eventing provides to enable loosely coupled event-driven architectures.
 This was created for a talk at the CNCF Toronto Q3 Meetup 2023.
+
+# Slide Deck
+
+You can find the presentation slide deck here: [Knative EDA Demos](https://docs.google.com/presentation/d/1Fi4jq1d7Tl1Q0VHkEjhPkdv-gfxPLIBqaeWod_dezvM/edit#slide=id.p)
```

Comment:
@Leo6Leo this doesn't seem to match the title of the talk we gave. Could you:
1. Re-name the slide deck to the title of the talk
2. Change the text for this link to the title of the talk as well?